src/components/timeline/TrackThread.js

```diff
@@ -125,45 +124,39 @@ class TimelineTrackThreadImpl extends PureComponent<Props> {
     event: SyntheticMouseEvent<>,
     sampleIndex: IndexIntoSamplesTable
   ) => {
-    const modifier = getTrackSelectionModifier(event);
-    switch (modifier) {
-      case 'none': {
-        const {
-          threadsKey,
-          selectLeafCallNode,
-          selectRootCallNode,
-          focusCallTree,
-          invertCallstack,
-          selectedThreadIndexes,
-        } = this.props;
+    const modifiers = getTrackSelectionModifiers(event);
```

Comment:
Here it looks like it's a lot of change, but if you look at the changes without the whitespace changes you'll see the code has just been reordered.